Application/src/model/Menu.java

```diff
@@ -28,6 +42,11 @@
     private static Button button_next;
     private static Button button_previous;
     private static Button button_auto;
+    private static Button button_join_lobby;
+    private static Button button_create_lobby;
+    private static Button button_remove_lobby;
+    private static Button button_import_lobby;
```

Comment:
import lobby sounds like you import the lobby itself, it's clearer below, though